tests/ChatTests.hs

```diff
@@ -37,30 +44,55 @@ testAddContact = describe "add chat contact" $
       Just inv' <- invitation <$> getWindow alice
       bob ##> ("/c " <> inv')
       concurrently_
-        (bob <## "alice_1 is connected")
-        (alice <## "bob_1 is connected")
+        (bob <## "alice_1 (Alice) is connected")
+        (alice <## "bob_1 (Bob) is connected")
       alice #> "@bob_1 hello"
       bob <# "alice_1> hello"
       bob #> "@alice_1 hi"
       alice <# "bob_1> hi"
       -- test deleting contact
       alice ##> "/d bob_1"
       alice <## "bob_1 is deleted"
-      chatCommand alice "@bob_1 hey"
+      alice #:> "@bob_1 hey"
       alice <## "no contact bob_1"
```

Comment:
might be an overkill, but maybe send one more message to `@bob` to verify he wasn't deleted?

Reply:
it is indeed :)